src/Components/Aspire.Oracle.EntityFrameworkCore.Database/AspireOracleEFCoreDatabaseExtensions.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Aspire;
+using Aspire.Oracle.EntityFrameworkCore.Database;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+using Oracle.EntityFrameworkCore;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods for configuring EntityFrameworkCore DbContext to Oracle database 
+/// </summary>
+public static class AspireOracleEFCoreDatabaseExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:Oracle:EntityFrameworkCore:Database";
```

Comment:
```suggestion
    private const string DefaultConfigSectionName = "Aspire:Oracle:EntityFrameworkCore";
```

Since that will be the name of the component.

Reply:
Done